components/Footer.tsx

```diff
@@ -94,26 +95,38 @@ export default function Footer() {
 
             <FooterSection title="Connect">
               <div className="flex space-x-4">
-                <SocialLink
-                  href={env.NEXT_PUBLIC_YOUTUBE_URL}
-                  icon={FaYoutube}
-                  label="YouTube"
-                />
-                <SocialLink
-                  href={env.NEXT_PUBLIC_LINKEDIN_URL}
-                  icon={FaLinkedin}
-                  label="LinkedIn"
-                />
-                <SocialLink
-                  href={env.NEXT_PUBLIC_GITHUB_URL}
-                  icon={FaGithub}
-                  label="GitHub"
-                />
-                <SocialLink
-                  href={`mailto:${env.NEXT_PUBLIC_CONTACT_EMAIL}`}
-                  icon={FaEnvelope}
-                  label="Email"
-                />
+                {env.NEXT_PUBLIC_YOUTUBE_URL && (
+                  <SocialLink
+                    href={env.NEXT_PUBLIC_YOUTUBE_URL}
+                    icon={FaYoutube}
+                    label="YouTube"
+                  />
+                )}
+                {
+                  <SocialLink
+                    href={env.NEXT_PUBLIC_LINKEDIN_URL}
+                    icon={FaLinkedin}
+                    label="LinkedIn"
+                  />
+                }
+                {
+                  <SocialLink
+                    href={env.NEXT_PUBLIC_GITHUB_URL}
+                    icon={FaGithub}
+                    label="GitHub"
+                  />
+                }
+                {
+                  <SocialLink
+                    href={`mailto:${env.NEXT_PUBLIC_CONTACT_EMAIL}`}
+                    icon={FaEnvelope}
+                    label="Email"
+                  />
+                }
+                {/* Static Twitter icon */}
+                <div className="text-gray-300 transition-colors duration-300 hover:text-[rgb(176,142,230)]">
+                  <BsTwitterX className="h-6 w-6" aria-label="Twitter" />
+                </div>
```

Comment:
1. Why is the twitter icon static? 
2. Why is there no link to it based on `env` value?
3. Let's not hardcode colors. Use colors that are already defined in the themes. This is so that colors are compatible with any theme configurations for different orgs.